src/Network/Network/AzureFirewall/NetworkRule/NewAzureFirewallNetworkRuleCommand.cs

```diff
@@ -87,13 +102,13 @@ public override void Execute()
             // Only one of DestinationAddress or DestinationFqdns is allowed
             if ((DestinationAddress != null) && (DestinationFqdn != null))
             {
-                throw new ArgumentException("Both DestinationAddress and DestinationFqdns not allowed");
+                throw new ArgumentException("Both DestinationAddress and DestinationFqdns not allowed.");
```

Comment:
I understand you didn't create this message but this line is confusing, can you put something like "only one of DestinationAddress or DestinationFqdns is allowed"?